terraform/modules/filecoin_node/forest.sh

```diff
@@ -85,18 +85,15 @@ if [ -n "${NEW_RELIC_API_KEY}" ] ; then
   NEW_RELIC_REGION="${NEW_RELIC_REGION}" \
   /usr/local/bin/newrelic install -y
 
+# The provided configurations are specific to New Relic. To gain a deeper understanding of these configuration details, you can visit:
+# https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#offline-time-to-reset
 cat >> /etc/newrelic-infra.yml <<EOF
-include_matching_metrics:
-  process.name:
-    - regex "^forest.*"
-    - regex "^fail2ban.*"
-    - regex "^rsyslog.*"
-    - regex "^syslog.*"
-    - regex "^gpg-agent.*"
 metrics_network_sample_rate: -1
-metrics_process_sample_rate: 300
-metrics_system_sample_rate: 300
-metrics_storage_sample_rate: 300
+metrics_process_sample_rate: -1
```

Comment:
Why do we discard Forest process metrics?

Reply:
Please elaborate. I don't see how metric of 1 process, scraped every 5 minutes or so, are able to produce several gigabytes of data in a month.

Reply:
What kind of data? I'd like us to understand this. I'm reluctant to believe that Linux generates so much data for a single process.